ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/RulerTask.kt

```diff
@@ -16,6 +16,7 @@
 
 package com.spotify.ruler.plugin
 
+import com.android.build.gradle.internal.dsl.BaseAppModuleExtension
```

Comment:
If we have to consume the extension at all (which I think we should try to avoid and extract the information by reading a configuration instead) then we should use the public `com.android.build.api.dsl.ApplicationExtension` instead of this internal API.